system-design/app-interoperability/application-package-definition.md

```diff
@@ -1,14 +1,25 @@
 # Application Package Definition
 
-This section addresses how an application is packaged by the “App Developer” who has implemented the application and aims to provide it to Margo-conformant systems. An application aggregates one or multiple [OCI Containers](https://github.com/opencontainers). The **application package** is made available in an [application registry](./workload-orch-to-app-reg-interaction.md).  
+This section addresses how an application is packaged by the “Application Developer” who has implemented the application and aims to provide it to Margo-conformant systems. An application aggregates one or more [OCI Containers](https://github.com/opencontainers). The **application package** is made available in an [application registry](./workload-orch-to-app-reg-interaction.md).  
 
-The application package comprises (a) the **application description** file `margo.yaml`, which contains informative contents and metadata (e.g., for application discovery purposes), (b) the **application manifests** ([Helm Chart](https://helm.sh/docs/) and [Docker Compose](https://github.com/compose-spec/compose-spec/blob/master/03-compose-file.md)) containing deployment instructions, and optionally (3) **application resources** (e.g., container binaries), which are necessary to enable the lifecycle of the application.
+The application package comprises:
 
-The application manifest SHALL be defined as a Helm Chart AND/OR a Docker Compose. If either one cannot be implemented it MAY be omitted.  
+- The **application description** file `margo.yaml`, which contains information about the application's marketing details (e.g., description, icon, release notes, license file, etc.), resource requirements, required input parameters, and application's installation packages (e.g,  Helm charts, docker-compose package) location.
```

Comment:
If we stop calling them (i.e. referenced Helm Charts and Docker Compose files) "application manifests" here, then we need to do that consistently through out this page (and the rest of the spec).

Reply:
This has been updated.